drivers/potentiometer/ad5258_59/ad5259.c

```diff
@@ -0,0 +1,455 @@
+/***************************************************************************//**
```

Comment:
Drop the `Formated for astyle` text from commit description and also there is a typo in there, `respeectively`.